apps/verticalhome/index.html

```diff
@@ -67,22 +69,23 @@
     <script defer src="js/more_apps_navigation.js"></script>
     <script defer src="js/clock.js"></script>
     <script defer src="js/app.js"></script>
-    <!-- <script defer src="shared/js/icc_helper.js"></script> -->
+        <!-- <script defer src="shared/js/icc_helper.js"></script> -->
     <!-- <script defer src="shared/js/version_helper.js"></script> -->
     <!-- <script defer src="js/configurator.js"></script> -->
   </head>
   <body class="theme-media">
     <div id="curtain"></div>
     <section id="main-screen" role="region">
+    <div id="main-screen-header" class="hhidden">Manage Homescreen</div>
 
       <div id="clock">
         <div id="clock-time"></div>
         <div id="date"></div>
       </div>
 
-      <div id="pinned-apps-list">
+      <div id="pinned-apps-list" class="">
```

Comment:
Remove 'class' attribute. 
I checked that your code works good without this attribute in index.html file.